frontend/src/scene/payment/PaymentView.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import styled from 'react-emotion';
+import Modal, { Title, Content as ModalContent } from '../../components/modal';
+import Button from '../../components/button';
+import posed from 'react-pose';
+import { connect } from 'utils';
+import PaymentState from './state';
+import qs from 'qs';
+
+const Content = styled(ModalContent)`
```

Comment:
This styling is unfit for this view

Reply:
Not done. You can update this after merging this